harness/determined/cli/experiment.py

```diff
@@ -605,7 +605,7 @@ def experiment_logs(args: Namespace) -> None:
             rank_ids=args.rank_ids,
             sources=args.sources,
             stdtypes=args.stdtypes,
-            min_level=args.level,
+            min_level=bindings.v1LogLevel.__members__[args.level],
```

Comment:
can just do `bindings.v1LogLevel[args.level]`. getting enum by name is built-in functionality in python's `enum`

Reply:
:themoreyouknow: Thanks!